cfme/physical/provider/redfish.py

```diff
@@ -64,3 +67,18 @@ def view_value_mapping(self):
             'name': self.name,
             'prov_type': 'Redfish'
         }
+
+
+@attr.s
+class RedfishPhysicalServer(PhysicalServer):
+
+    INVENTORY_TO_MATCH = ['power_state']
+    STATS_TO_MATCH = ['cores_capacity', 'memory_capacity']
+
+    def __init__(self):
```

Comment:
There's no definition of `__init__` when using `@attr.s`, this should not be included.

Reply:
Extraneous `__init__` removed.